### DISCUSSION TITLE
[@types/async-retry] Argument of type '{ retries: number; }' is not assignable to parameter of type 'Options'.

### DISCUSSION THREAD

**Discussion by mattsputnikdigital** [DefinitelyTyped/DefinitelyTyped#62551] [Issues with a @types package] [answered]
async-retry has not updated since 1.3.3 Aug 2021 but the types package seems to have been updated since and now seems broken.

https://github.com/vercel/async-retry/issues/97

```
Argument of type '{ retries: number; }' is not assignable to parameter of type 'Options'.
  Object literal may only specify known properties, and 'retries' does not exist in type 'Options'.  TS2345

    37 |     },
    38 |     {
  > 39 |       retries: RETRIES,
       |       ^
    40 |     }
    41 |   );
```

[a comment by an automated account is not shown]

**Comment by BendingBender** [comment]
@mattsputnikdigital Can you please provide a full example of the code that doesn't work for you? Just FIY, your use case is actually tested, so it should work: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/async-retry/async-retry-tests.ts#L17 .

I suspect some kind of a dependency problem with your setup. You can try to solve it by simply deleting your `package-lock.json`/`yarn.lock`.

**Comment by mattsputnikdigital** [comment]
Im bumping the site from Typescript 3 to 4 so this may well have something to do with it. Im locking the package at 1.3 as that currently works.

My code is pretty much an exact copy of the example and has been fine and working for 2 years. If once TS is updated, and im still having the error, I will update this

**Comment by mattsputnikdigital** [comment, marked as answer]
@BendingBender updating to TS 4 fixed the issue.

**Reply by BendingBender** [reply]
nice
